StripePaymentSheet/StripePaymentSheet/PaymentSheet/STPAnalyticsClient+PaymentSheet.swift

```diff
@@ -48,21 +50,25 @@ extension STPAnalyticsClient {
             ),
             duration: AnalyticsHelper.shared.getDuration(for: .checkout),
             linkEnabled: linkEnabled,
-            activeLinkSession: activeLinkSession
+            activeLinkSession: activeLinkSession,
+            currency: currency,
+            params: ["payment_method": paymentOption.name.lowercased()]
```

Comment:
Does this include a card number, or is it just the generic payment method type?

Reply:
Just generic, so `card` for card and `us bank account` for bank acct.